src/views/RenderView.tsx

```diff
@@ -239,11 +240,11 @@ export default function RenderView(
                 seg.geometry.attributes.position.setXYZ(0,-startPos.x,startPos.z,startPos.y); //Vicon coord conversion is -x,z,y
                 seg.geometry.attributes.position.setXYZ(1,-endPos.x,endPos.z,endPos.y);
                 seg.geometry.attributes.position.needsUpdate = true;
-                seg.visible = true;
+                seg.visible = showSegments;
```

Comment:
if it's not visible, couldn't we skip updating the position too?

Reply:
fixed